packages/now-static-build/src/index.ts

```diff
@@ -231,12 +232,8 @@ export async function build({
         try {
           await timeout(
             new Promise(resolve => {
-              const checkForPort = (data: string) => {
```

Comment:
I took the easy to just replace the checking, but yes in fact I should have done something cleaner.
I'm not sure if its useful to use a loop, because the isPortReachable can take a timeout duration.

Meaning, I could really simplify the code with 

```js
try{
    isPortReachable(devPort, 5 * 60 * 1000);
} catch (err) {
    throw new Error(
        `Failed to detect a server running on port ${devPort}.\nDetails: https://err.sh/zeit/now/now-static-build-failed-to-detect-a-server`
    );
}
```

Reply:
@TooTallNate In which cases and how are they problematic ?